pip/req/req_set.py

```diff
@@ -447,6 +474,12 @@ def _prepare_file(self, finder, req_to_install):
                 if self.is_download:
                     req_to_install.archive(self.download_dir)
             elif req_to_install.satisfied_by:
+                if require_hashes:
+                    logger.info(
+                        'Since it is already installed, we are trusting this '
+                        'package without checking its hash. To ensure a '
+                        'completely repeatable environment, install into an '
+                        'empty virtualenv.')
```

Comment:
Do we want this to be ``INFO`` or ``DEBUG``? I think it's going to be a lot of noise if you do ``pip install -r requirements.txt`` just to sync an updated requirements.txt with an existing virtual environment.